src/core_plugins/region_map/public/region_map_vis_params.js

```diff
@@ -15,27 +15,29 @@ uiModules.get('kibana/region_map')
 
         $scope.collections = $scope.vis.type.editorConfig.collections;
 
+        if ($scope.vis.params.selectedLayer && !$scope.vis.params.selectedLayer._id) {
+          //tracking by id wasn't ideal.
```

Comment:
when will this happen ? it seems that _id will always be set, as selectedLayer can only be one of vectorLayers which was created using `mapToLayerWithID` which will always generate an id ... (`.` in worse case if name was not set ?)

Reply:
then we don't need the first parameter at all since it seems its only used here ?

Reply:
i removed this (see comment above)